bin/zowe-setup-certificates.env

```diff
@@ -23,6 +23,12 @@ EXTERNAL_CERTIFICATE_AUTHORITIES=
 # optional - Public certificates of z/OSMF - multiple certificates 
 # delimited with space has to be enclosed with quotes ("cer1 cer2")
 ZOSMF_CERTIFICATE=
+# Fallback to native authentication with ZSS auth/session tokens if JWT authentication fails
```

Comment:
Can this comment be a bit more open-ended for future use:
`If SSO token not present, zowe components are allowed to attempt authentication with other user-provided data`